.github/workflows/deploy-dev.yml

```diff
@@ -0,0 +1,34 @@
+name: Dev-CD
+
+on:
+  push:
+    branches:
+      - develop
+
+env:
+  SURGE_LOGIN: ${{ secrets.SURGE_LOGIN }}
+  SURGE_TOKEN: ${{ secrets.SURGE_TOKEN }}
+  NODE_VERSION: "10.x"
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Install Packages
+        run: yarn install
+
+      - name: Run Tests
+        run: yarn test
+
+      - name: Build Project
+        run: yarn build
+
+      - name: "Deploy to Surge"
+        run: surge --project ./build --domain blazehub-prod.surge.sh
```

Comment:
I was expecting you to deploy to `blazehub-staging.surge.sh` or so since this is the dev branch.